src/drivers/imx/esai.c

```diff
@@ -398,6 +398,18 @@ static int esai_get_fifo(struct dai *dai, int direction, int stream_id)
 	}
 }
 
+static uint32_t esai_get_fifo_depth(struct dai *dai, int direction)
+{
+	switch (direction) {
+	case DAI_DIR_PLAYBACK:
+	case DAI_DIR_CAPTURE:
+		return dai->plat_data.fifo[direction].depth;
+	default:
+		dai_err(dai, "esai_get_fifo_depth(): Invalid direction");
+		return -EINVAL;
```

Comment:
the function returns an unsigned type...

Reply:
yeah